keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -97,14 +114,47 @@ Hopefully the content previously contained in [umbrella issues][] will be tracke
 
 ## Implementation History
 
-## Drawbacks [optional]
+## Drawbacks
+
+The biggest drawback is that this is yet another thing to keep running.
+Currently the markdown files are workable but not super discoverable.
+However, they utilize the familiar mechanisms and do not require extra effort or understanding to publish.
+
+The current mechanisms also scale across orgs and repos.
+This is a strength as the information is close to the code but also a big disadvantage as it ends up being much less discoverable.
+
+## Alternatives
+
+One alternative is to do nothing.
+However, the smattering of markdown through many repos is not scaling and is not discoverable via Google or for other members of the community.
+
+The main alternative here is to build something that is integrated into the user facing kubernetes.io site.
+This is not preferred for a variety of reasons.
+
+* **Workflow.** Currently there is quite a bit of process for getting things merged into the main site.
+  That process involves approval from someone on SIG-Docs from an editorial point of view along with approval for technical accuracy.
+  The two stage approval slows down contributions and creates a much larger barrier than the current markdown based flow.
+  In addition, SIG-Docs is already stretched thin dealing with the (more important) user facing content that is their main charter.
+* **Quality standards.** The bar for the user facing site is higher than that of the community site.
+  Speed and openness of communication dominates for the community facing site.
```

Comment:
Docs maintainers are also currently in the practice of judging just how much we need to meet arbitrary quality standards. We try to err on the side of getting useful content into the docs. IOW, we're flexible on this. We don't unilaterally impose an arbitrary set of content standards.

Reply:
It would be helpful to consistently talk about a contributor site, rather than using "contributors" and "community" interchangeably. It's clear what contributors are, and why a site matters for them; it's less clear what a community is, and site needs even less so.

Reply:
I’m happy to center this around “contributors” if that helps makes progress.  The reason that they are somewhat conflated here is that I see this as an extension and evolution of the “community” repo.  If this repo were called “kubernetes/contributors” then I’m guessing we’d be more consistent.